src/Containers/SearchBarContainer.php

```diff
@@ -12,10 +13,17 @@ class SearchBarContainer
 {
     /**
      * @param Twig $twig
+     * @param SearchService $searchService
      * @return string
      */
-    public function call(Twig $twig):string
+    public function call(Twig $twig, SearchService $searchService):string
     {
+        if (!$searchService->isAliveTest()) {
```

Comment:
I click tested this real quick cause it would be strange that this doesn't work without an alive test since this template doesn't need any data from our API.
Turns out that this alive check is indeed not necessary. Please apply all changes mentioned in this review and check if you can confirm my click test result.